translator/trace/jaeger/traces_to_jaegerproto.go

```diff
@@ -115,42 +115,64 @@ func resourceToJaegerProtoProcess(resource pdata.Resource) *model.Process {
 	}
 
 	process := model.Process{}
+	attrsCount := attrs.Cap()
 	if serviceName, ok := attrs.Get(conventions.AttributeServiceName); ok {
 		process.ServiceName = serviceName.StringVal()
-		attrs.Delete(conventions.AttributeServiceName)
+		attrsCount--
```

Comment:
Please fix the bug in a separate PR since this is very confusing now.

Reply:
Submitter another PR https://github.com/open-telemetry/opentelemetry-collector/pull/907, please take a look, I'll rebase once it's merged